nodejs/awsx/ecs/container.ts

```diff
@@ -270,6 +270,30 @@ export interface ContainerDependency {
     containerName: pulumi.Input<string>;
 }
 
+/**
+ * See https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_FirelensConfiguration.html
+ * for more details
+ */
+export interface FirelensConfiguration {
+    /**
+     * The options to use when configuring the log router. This field is optional and can be used to specify a custom
+     * configuration file or to add additional metadata, such as the task, task definition, cluster, and container
+     * instance details to the log event.
+     */
+    options?: pulumi.Input<FirelensConfigurationOptions>;
+
+    /**
+     * The log router to use.
+     */
+    type: pulumi.Input<"fluentd" | "fluentbit">;
+}
+
+export interface FirelensConfigurationOptions {
+    enableEcsLogMetaData?: pulumi.Input<true | false>;
```

Comment:
is this not `Input<boolean>`?

Reply:
d'oh! Complete brain fail :) Fixed - thanks for picking that up!